libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -441,6 +438,20 @@ impl Greenlight {
         Ok(on_chain_balance)
     }
 
+    async fn pending_onchain_balance(&self, funds: cln::ListfundsResponse) -> Result<u64> {
+        let pending_onchain_balance = funds.outputs.iter().fold(0, |a, b| {
+            if b.status != 0 || !b.reserved {
```

Comment:
Can you explain the logic here? It looks like we consider utxos that are confirmed or not reserved as pending balance.

Reply:
This logic was flawed. Since the channel is not closed yet then there is not much help in using the output of list funds. Instead we need to go thought all of the channels that are closed pending and sum up `our_amount_msat` for each channel.